Automattic/pocket-casts-ios
# Use S3 for fastlane match

What says in the title.

Google Cloud API is no longer used and is now removed from the .configure file.

## To test

You can run `bundle exec fastlane ios configure_code_signing` to verify code sign things can be fetched and installed correctly.

## Checklist
